src/app/components/subtopics-list/subtopics-list.component.html

```diff
@@ -0,0 +1,9 @@
+<app-search-section
+  *ngIf="topicName"
```

Comment:
Dare to use the new directives for *ngIf and *ngFor

Reply:
I tried but for some reason, I don't know it doesn't work. You can also see here that I commented on the code with the attempt.
![image](https://github.com/radumaftei/frontend-learning-app/assets/100227050/9e6e8dd1-b0bc-4516-8b70-3dd2c64b7cdd)